File: 8kyu/what-is-between.js
//Describtion:

//my Solution:
function between(a, b) {
    return Array.from(new Array(b - a + 1), x => a++)
  }

//Other solutions:
function between(a, b) {
    // your code here
    arr = []
    for(i = a;i <= b; i++){ 
        arr.push(i)
    }
    return arr
  } 

File: Python/8kyu/what-is-between.py
#Describtion:
# Complete the function that takes two integers (a, b, where a < b) and return an array of all integers between the input parameters, including them.

# For example:

# a = 1
# b = 4
# --> [1, 2, 3, 4]

#Solution:
def between(a, b):
    return list(range(a, b+1))

#Other solutions:
def between(a,b):
    return [result for result in range(a,b+1)]